Resources/Prototypes/_EE/Entities/Clothing/SolAlliance/SAN/Modsuits/CSA-85x.yml

```
@@ -0,0 +1,192 @@
+        params:
+          volume: -5
+      requiresWorn: true
+      distanceWalking: 2
+      distanceSprinting: 3
+
+- type: entity
+  parent: [ClothingModsuitBootsStandard, SolAllianceInfo]
+  id: ClothingModsuitBootsSANMarine
+  name: CSA-85x "Mazu" modsuit boots
+  description: A special modular suit boots designed for compact folding inside modular suit control.
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)